include/vrv/view.h

```diff
@@ -480,6 +483,7 @@ class View {
     ///@{
     void DrawVerticalLine(DeviceContext *dc, int y1, int y2, int x1, int width, int dashLength = 0);
     void DrawHorizontalLine(DeviceContext *dc, int x1, int x2, int y1, int width, int dashLength = 0);
+    void DrawRoundedLine(DeviceContext *dc, int x1, int y1, int x2, int y2, int width);
```

Comment:
Why rounded? It is just a line, isnt' it? We would at least need a radius parameter.

Reply:
It's a line with rounded end points.